sys/src/9/port/devramfs.c

```diff
@@ -208,19 +273,44 @@ ramread(Chan *c, void *buf, int32_t n, int64_t off)
 	return n;
 }
 
+typedef double Align;
+typedef union Header Header;
+
+union Header {
+        struct {
+                Header* next;
+                uint    size;
+        } s;
+        Align   al;
```

Comment:
This is copied from qmalloc.c so I could get the header. I'll move the qmalloc one into a suitable header file.